contracts/SinglePlayerCommit.sol

```diff
@@ -51,213 +61,470 @@ contract SinglePlayerCommit is Ownable {
     event CommitmentEnded(address committer, bool met, uint256 amountPenalized);
     event Deposit(address committer, uint256 amount);
```

Comment:
efficiency police: do we need a Deposit event? Could we get the same information from the event emitted by the token transfer?

Reply:
Created issue #16